doc/tutorials/python-kdb.md

```diff
@@ -0,0 +1,84 @@
+# How-To: Python kdb
+
+## Introduction
+
+When programming in python it is possible to access the kdb database, changing values of existing keys, adding and deleting keys and a few other things.
+
+## First Steps
+
+In order to being able to use kdb, obviously, you at first need to `import kdb`. You need access to an python-object of kdb. This is accomlished by calling `kdb.KDB()` and saving this to a variable because later on this object will be needed for various operations. 
+The easiest way to do this would be:
+```
+import kdb
+with kdb.KDB() as k:
+	print 'Hello world! I have a kdb-instance! :D'
+	# do all kinds of operations explained below
+```
+
+## Keyset
+
+A keyset is basically a list of the keys that lie within the specified range. At first it is necessary to create a new keyset. When simply calling `kdb.KeySet()` the keyset is of size 0. There is no restriction to the keyest's size. It is possible to specify a certain (maximum) size for a keyset. To load keys into to keyset from the database you simply call the method `get` provided by the kdb-object.
+
+```
+import kdb
+with kdb.KDB() as k:
+	# create empty keyset
+	ks = kdb.KeySet()
+	print len(ks) # should be 0
+	# load an existing keyset
+	k.get(ks, 'path/to/keys')
```

Comment:
please always prefix pathes `/` (because it is the most common case).

Maybe also bring an example how to write something so that users are not only getting empty keysets ;)